sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -1085,3 +1074,25 @@ func withDynamicResolver(ctx context.Context, ref string, registryConfig *Regist
 		return defaultResolver
 	}
 }
+
+// withMaskedPaths sets an alternate list of masked paths
+func withMaskedPaths() oci.SpecOpts {
```

Comment:
I think this function should let the caller pass what paths to unmask, that way the function is more usable in other places if we need to unmask depending on different conditions.


```suggestion
func withMaskedPaths(unmaskedPaths []string) oci.SpecOpts {
```

Reply:
I was originally going to, but I thought we might want to be purposefully inflexible given the nature of masked paths.